setup-env

```diff
@@ -251,7 +251,7 @@ for req_file in "requirements-dev.txt" "requirements-test.txt" "requirements.txt
 done
 
 # Install all necessary mypy type stubs
-mypy --install-types src/
+mypy --install-types --non-interactive src/
```

Comment:
Can this change be added to [cisagov/skeleton-generic](https://github.com/cisagov/skeleton-generic)?  Also, does the command return a zero return code with the addition of this flag?  (This is a problem I ran into just this morning.)

Reply:
I agree this should be pushed up to the skeleton. 

I confirmed  command does return with exit `0` by printing the output in `setup-env`. 

```console
...
Installing missing stub packages:
~/.pyenv/versions/cyhy-config/bin/python3 -m pip install types-requests

Collecting types-requests
  Downloading types_requests-2.32.0.20240914-py3-none-any.whl.metadata (1.9 kB)
Requirement already satisfied: urllib3>=2 in ~/.pyenv/versions/cyhy-config/lib/python3.12/site-packages (from types-requests) (2.2.3)
Downloading types_requests-2.32.0.20240914-py3-none-any.whl (15 kB)
Installing collected packages: types-requests
Successfully installed types-requests-2.32.0.20240914

Success: no issues found in 3 source files
mypy exit code: 0
...
```

Reply:
This code actually comes from [cisagov/skeleton-python-library](https://github.com/cisagov/skeleton-python-library) (not [cisagov/skeleton-generic](https://github.com/cisagov/skeleton-generic) as I thought) so I created cisagov/skeleton-python-library#138.